qemu/tests/cfg/win_driver_base.cfg

```diff
@@ -0,0 +1,96 @@
+            device_name = "VirtIO Balloon Driver"
+            driver_id_pattern = "(.*?):.*?VirtIO Balloon Driver"
+            sub_test = balloon_check
+            virtio_balloon_pause = 10.0
+            free_mem_cmd = cat /proc/meminfo |grep MemFree
+            ratio = 0.5
+            run_sub_test_after_balloon = no
+            test_tags = "evict enlarge"
+            balloon_type_evict = evict
+            balloon_type_enlarge = enlarge
+        - with_netkvm:
+            driver_name = netkvm
+            nics += " nic2"
+            nic_model_nic1 = virtio
+            nic_model_nic2 = rtl8139
+            device_name = "Red Hat VirtIO Ethernet Adapter"
+            hostpassword = redhat
+            sub_test = netperf_stress
+            netperf_link = netperf-2.6.0.tar.bz2
+            server_path = /var/tmp/
+            client_path = /var/tmp/
+            test_protocols = TCP_STREAM
+            netperf_client = ${vms}
+            netperf_server = localhost
+            netperf_test_duration = 200
+            wait_before_subtest = 100
+            ping_ext_host = "yes"
+            ext_host_get_cmd = "ip route | awk '/default/ { print $3 }'"
+            shell_client = "nc"
+        - with_viostor:
+            driver_name = viostor
+            device_name = "Red Hat VirtIO SCSI controller"
+            sub_test = iozone_windows
+            drive_format_image1 = ide
+            images += " stg"
+            image_name_stg = "images/storage"
+            image_size_stg = 4G
+            drive_format_stg = virtio
+            disk_letter = I
+            disk_index = 1
+            format_disk = yes
+            force_create_image_stg = yes
+            remove_image_stg = yes
+            driver_id_pattern = "(.*?):.*?VirtIO SCSI Disk Device"
+            iozone_cmd = "WIN_UTILS:\Iozone\iozone.exe -azR -r 64k -n 1G -g 2G -M -i 0 -i 1 -b iozone.xls -f ${disk_letter}:\testfile"
+            iozone_timeout = 7200
+            wait_bg_time = 180
+        - with_vioscsi:
+            driver_name = vioscsi
+            device_name = "Red Hat VirtIO SCSI pass-through controller"
+            drive_format_image1 = ide
+            images += " stg"
+            image_name_stg = "images/storage"
+            image_size_stg = 4G
+            drive_format_stg = scsi-hd
+            disk_letter = I
+            disk_index = 1
+            format_disk = yes
+            force_create_image_stg = yes
+            remove_image_stg = yes
+            driver_id_pattern = "(.*?):.*?VirtIO SCSI pass-through controller"
+            iozone_cmd = "WIN_UTILS:\Iozone\iozone.exe -azR -r 64k -n 1G -g 2G -M -i 0 -i 1 -b iozone.xls -f ${disk_letter}:\testfile"
+            iozone_timeout = 7200
+            wait_bg_time = 180
+    variants:
+        - with_uninstall:
+            uninstall = True
+        - with_downgrade:
+            uninstall = False
+            cdrom_virtio_downgrade = isos/windows/virtio-win-downgrade.iso
```

Comment:
Will remove downgrade and uninstall from case "single_driver_update" when this case is merged.